src/components/organisms/UserProfileModal/UserProfileModal.scss

```diff
@@ -62,6 +62,15 @@ $modal-max-width: 540px;
     .profile-text {
       grid-column: 2;
       grid-row: 1;
+
+      &__venueId {
```

Comment:
```suggestion
      &__recent-venue {
```

Reply:
Why. It contains venue id. I think that is right name for the block.

Reply:
https://github.com/sparkletown/sparkle/pull/1484/commits/315d06f92e49515c4b4f1c6603384c2e37eba7e7